frontend/src/components/Navbar/Navbar.tsx

```diff
@@ -19,37 +23,91 @@ function Navbar() {
-      </div>
-      <div>
-        <button className={classes.hover} onClick={toggleCredits}>
-          Credits
-        </button>
-        <div onClick={toggleDropDown}>
-          <ProfileIcon url="/yellowshirt.svg"/>
-          {showDropDown && <ProfileDropdown username='Chris'/>}
+    <div>
+      <nav className={classes.navbar}>
+        <div>
+          <button
+            className={`${classes.hover} ${classes.logo}`}
+            onClick={() => handleNavigation("/home")}
+          >
+            <Logo size="lg" />
+          </button>
+          <button
+            className={`${classes.hover} ${classes.hideOnMobile}`}
+            onClick={() => navigate("/leaderboard")}
+          >
+            Leaderboard
+          </button>
+          <button
+            className={`${classes.hover} ${classes.hideOnMobile}`}
+            onClick={() => navigate("/gamemodes")}
+          >
+            Gamemodes
+          </button>
+          <button className={`${classes.hover} ${classes.hideOnMobile}`}>
+            Help
+          </button>
         </div>
-      </div>
-      {showCredits &&
-        createPortal(
-          <Credits onClick={toggleCredits} />,
-          document.getElementById("overlay-root") as HTMLElement,
-        )}
-    </nav>
+
+        <div>
+          <button
+            className={`${classes.hover} ${classes.hideOnMobile}`}
+            onClick={toggleCredits}
+          >
+            Credits
+          </button>
+          <div
+            onClick={() => setShowProfileDropDown(!showProfileDropDown)}
+            className={classes.hideOnMobile}
+          >
+            <ProfileIcon url="/yellowshirt.svg" />
+            {showProfileDropDown && <ProfileDropdown username="Chris" />}
+          </div>
+        </div>
+        {showCredits &&
+          createPortal(
+            <Credits onClick={toggleCredits} />,
+            document.getElementById("overlay-root") as HTMLElement,
+          )}
+        <button
+          className={classes.hamburger}
+          onClick={() => setShowDropDown(!showDropDown)}
+        >
+          {showDropDown ? <X color="white" /> : <Menu color="white" />}
+        </button>
+      </nav>
+      {showDropDown && (
+        <Sheet dropdownNavbar className={classes.sheet}>
+          <button onClick={() => handleNavigation("/profile")}>
+            <User className={classes.userIcon} color="hsl(52, 100%, 50%)" />
+            Profile
+          </button>
+          <button onClick={() => handleNavigation("/gamemodes")}>
+            📌Gamemodes
+          </button>
+          <button onClick={() => handleNavigation("/leaderboard")}>
+            🏆Leaderboard
+          </button>
+          <button onClick={() => {
+            if (showCredits) {
+              setShowDropDown(false);
+              return;
+            } 
+            toggleCredits();
+            setShowDropDown(false);
+          }}>
+            💛Credits
```

Comment:
formatting isn't ideal - would be good if they were all icons and had some more padding between the text and the icons.

Reply:
honestly if you can't be bothered just get rid of the emojis

Reply:
sorry bad word choice - I meant layout